pkg/util/runner.go

```diff
@@ -33,6 +33,7 @@ func (r RealRunner) Run(command string, args ...string) error {
 	fmt.Println(fmt.Sprintf("Provisioning OpenShift via '%s %s'", command, args))
 	cmd := exec.Command(command, args...)
 	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stdout
```

Comment:
This will make sure if we pass any unsupported flag to oc like http proxy not support less than version 1.5.0 then it will tell user unsupported option. As per doc https://golang.org/pkg/os/exec/

```
        // If Stdout and Stderr are the same writer, at most one
        // goroutine at a time will call Write.
        Stdout io.Writer
        Stderr io.Writer
```